lib/features/give/widgets/choose_category.dart

```diff
@@ -0,0 +1,196 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/give/widgets/widgets.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+
+class ChooseCategory extends StatelessWidget {
+  const ChooseCategory({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.sizeOf(context);
+    final locals = context.l10n;
+
+    return ConstrainedBox(
+      constraints: BoxConstraints.expand(
+        width: size.width,
+        height: size.height,
+      ),
+      child: Column(
+        children: [
+          SizedBox(
+            height: size.height * 0.05,
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Expanded(
+                child: _buildSearchSeeAllCard(
+                  context,
+                  title: locals.discoverSearchButton,
+                  icon: FontAwesomeIcons.magnifyingGlass,
+                  onPressed: () => context.goNamed(
+                    Pages.chooseCategoryList.name,
+                    extra: CollecGroupType.none.index,
+                  ),
+                ),
+              ),
+              Expanded(
+                child: _buildSearchSeeAllCard(
+                  context,
+                  title: locals.discoverDiscoverButton,
+                  icon: FontAwesomeIcons.compass,
+                  onPressed: () => context.goNamed(
+                    Pages.chooseCategoryList.name,
+                  ),
+                ),
+              ),
+            ],
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Expanded(
+                child: FilterSuggestionCard(
+                  isFocused: false,
+                  color: CollecGroupType.charities.color,
+                  title: _getCategoryName(
+                    type: CollecGroupType.charities,
+                    context: context,
+                  ),
+                  activeIcon: CollecGroupType.charities.activeIcon,
+                  icon: CollecGroupType.charities.icon,
+                  width: size.width * 0.4,
```

Comment:
Not sure if this was existing code, but we should make use of the flex layouts (*column, row, expanded, margin, etc). This is *0.3/0.4 for width, heights are not scalable enough for all screen sizes

Reply:
Just tested on an iPhone SE and it's looks ok. But indeed I would have to change it in the end